Mage.Sets/src/mage/cards/m/MyrCustodian.java

```diff
@@ -0,0 +1,68 @@
+package mage.cards.m;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.keyword.ScryEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.game.Game;
+import mage.players.Player;
+
+import java.util.UUID;
+
+public class MyrCustodian extends CardImpl {
+    public MyrCustodian(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT, CardType.CREATURE}, "{3}");
+        this.addSubType(SubType.MYR);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(3);
+
+        //When Myr Custodian enters the battlefield, scry 2. Then each opponent may scry 1.
+        EntersBattlefieldTriggeredAbility entersBattlefieldTriggeredAbility =
+                new EntersBattlefieldTriggeredAbility(new ScryEffect(2));
```

Comment:
The scry effect here should have no hint, so just use `new ScryEffect(2, false)` to disable that

Reply:
done